packages/lib/src/components/FeedbackWidget/FeedbackWidget.tsx

```diff
@@ -128,8 +127,8 @@ export function FeedbackWidget({
     setValue('email', currentUser?.email || '')
     setValue('feedbackType', openInput || OpenInput.Other)
     setValue('screenshot', screenshot || '')
-    setValue('path', router.asPath || '')
-  }, [currentUser, openInput, screenshot, router.asPath, setValue])
+    setValue('path', window.location.pathname || '')
+  }, [currentUser, openInput, screenshot, setValue])
```

Comment:
We should use [`usePathname`](https://nextjs.org/docs/app/api-reference/functions/use-pathname) here.